src/Apps/Artwork/Components/PrivateArtwork/__tests__/PrivateArtworkAboutWork.jest.tsx

```diff
@@ -0,0 +1,30 @@
+import { screen } from "@testing-library/react"
+import { PrivateArtworkAboutWork } from "Apps/Artwork/Components/PrivateArtwork/PrivateArtworkAboutWork"
+import { setupTestWrapperTL } from "DevTools/setupTestWrapper"
+import { graphql } from "react-relay"
+import { PrivateArtworkAboutWorkQuery } from "__generated__/PrivateArtworkAboutWork_artwork.graphql"
+
```

Comment:
I suspect that I created this by accident because I keep getting errors when I try to import, but I don't know where this file is coming from?